1223-curator_worklist/1252-curator_expands_an_item_in_the_worklist.feature

```diff
@@ -1,6 +1,6 @@
     @1252
-    Scenario Outline: Curator expands an item in the Worklist
-        Given that a Curator views My Worklist tabs
+    Scenario: Curator expands an item in the Worklist
+        Given that a Curator views DOI Request tab (Part II)

```

Comment:
Line 7: there are many messages in a dialog box maybe?

Reply:
Updated now